app/controllers/admin/admin_controller.rb

```diff
@@ -1,3 +1,19 @@
 class Admin::AdminController < ApplicationController
   layout "admin"
+
+  def admin_title(model=nil, keys = [])
+    return t('.index_title') unless model.present?
```

Comment:
sorry, @veganstraightedge, I was using "lazy" in the "too lazy to type the whole path" sense, not "lazy" in the computer science/programming sense.

So I added a bunch of translations that look like this
```yaml
  admin:
    title_prepend: "Admin"
    articles:
      index_title: "Articles"
      show_title: "Articles : \"%{title}\""
      edit_title: "Editing article %{id} \"%{title} : %{subtitle}\""
```

one way of getting at the `index_title` in the `yaml` would be:

```ruby
t('admin.articles.index_title')
```

those paths can get pretty nested though, so rails gives you a special notation where if your translations are organized the same as your controller (or views) you can do the same lookup like this  (note the dot):

```ruby
t('.index_title')
```

this works here because the method is in the admin controller